MetaMask/metamask-extension
# Adding `FormTextField` component

## Explanation

Adding `FormTextField` that bundles `TextField`, `Label` and `HelpText` components together to create forms

* Fixes #16152 

## Screenshots/Screencaps

### After

https://user-images.githubusercontent.com/8112138/201794011-01ddafda-8374-4261-b983-1da18e0bb301.mov

## Manual Testing Steps

- Go to the latest build of Storybook on this PR
- Search `FormTextField` in the search bar
- Click on the result
- Check story and controls
- Check MDX docs
- Check console for any errros

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
